### PR TITLE
Added short links for all tools

### PR DESCRIPTION
**Pull request by KrauseFx**
Added rake task to automatically generate those files

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  tools = GEMS # we don't care about RAILS, since they're not mono repo
+  puts "Generating short links for: \n#{tools.join(', ')}"
+
+  tools.each do |current|
+    FileUtils.mkdir(current) unless File.directory?(current)
+
+    index = "
+            <script type=\"text/javascript\">
+              location.href = \"https://github.com/fastlane/fastlane/tree/master/#{current}\"
+            </script>"
```
Comment (mfurtak) [inline Rakefile]: minor suggestion - use a HEREDOC to get rid of the leading spaces and not need to escape quotes

File: Rakefile [hunk 2]
```diff
@@ -0,0 +1,19 @@
+task :generate do
+  require 'open-uri'
+  require 'fileutils'
+  content = open("https://raw.githubusercontent.com/fastlane/fastlane/master/Rakefile").read
+  content = content.split("desc").first # only the first lines are relevant
+  eval(content)
```
Comment (mfurtak) [inline Rakefile]: It's risky to evaluate code that is downloaded blindly from the Internet. :sweat_smile:

**Comment by mfurtak** [comment]
Looks pretty good. I left one small suggestion and one note about a concern. :+1:

File: Rakefile (hunk 2 above)
Comment (KrauseFx) [inline Rakefile]: Agreed. What do you suggest? Should we maintain yet another list of tools? Or is it time to have a REST API for a list of all tools, so we can access it from our back-end services and tools?

Comment (mfurtak) [inline Rakefile]: Hm, the REST API is interesting, but maybe overkill for a script like this?

It is not satisfying, but I think listing the tools again is reasonable. If we remember to come back here to use this script to generate another short link for a new tool, we can remember to update the local list of tools.

Obviously this is all just a suggestion so go with whatever feels right to you :+1:

**Comment by KrauseFx** [comment]
@mfurtak Updated the PR, let me know what you think :+1: Also, using fastlane instead of rake now

File: fastlane/Fastfile [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  tools = %w(fastlane fastlane_core deliver snapshot frameit pem sigh produce cert gym pilot credentials_manager spaceship scan supply watchbuild match screengrab)
+  puts "Generating short links for: \n#{tools.join(', ')}"
+
+  Dir.chdir("..") do
+    tools.each do |current|
+      FileUtils.mkdir(current) unless File.directory?(current)
+
+      index = <<-eos
+  <script type=\"text/javascript\">
+    location.href = \"https://github.com/fastlane/fastlane/tree/master/#{current}\"
```
Comment (mfurtak) [inline fastlane/Fastfile]: I believe you can remove the \-escaping for the quotes in here too, if you like! :+1:

**Comment by mfurtak** [comment]
Looks great! Ship it whenever you're ready :+1:

**Pull request merged**